getting-started-5.html

```diff
@@ -57,32 +57,69 @@ <h2><a class="anchor" id="installing"></a>Installing</h2>
 
 </li>
 
-<li><p class="first">Expand OMERO.insight client archive and move it to your home directory.</p>
+<div class="note">
+	<p class="note-text"><z class="bold">Windows</z>
+	<p class="note-text">Since version 5.5.0, OMERO.insight has two installers, <z class="bold">.msi</z> and <z class="bold">.exe</z>.</p>
+	<p class="note-text">The <z class="bold">.msi</z> installer will deploy the application in the <z class="bold">Program Files</z> folder of Windows and OMERO.insight is available to all the users of the target machine.</p>
+	<p class="note-text">The <z class="bold">.exe</z> file installs by default in the <z class="bold">userspace apps folder</z>. This can be changed during install, see screenshots in Step 2. below. OMERO.insight installed in the userspace is only available to the user who did the installation.</p>
+	<p class="note-text">A <z class="bold">Desktop icon</z> and a new OMERO.insight <z class="bold">Start menu item</z> will be created.</p>
+	</div><!-- End note -->
+
+<li><p class="first"><z class="bold">Windows:</z> Click onto the downloaded <z class="bold">.exe</z> or <z class="bold">.msi</z> file. This will run the installer.</p>
 
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<p class="bold">Windows:</p>
+<img alt="../images/gettingStartedWinSelectDest.png" src="images/gettingStartedWinSelectDest.png" style="width: 92%; class="align-center" " />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+
+<img alt="../images/gettingStartedWinWizard.png" src="images/gettingStartedWinWizard.png" style="width: 92%; class="align-center" " />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+
+<div class="note">
+	<p class="note-title">Mac</p>
+	<p class="note-text">Since version 5.5.0 OMERO.insight has a <z class="bold">.dmg</z> installer which will deploy OMERO.insight and open a new window with three icons.</p>
```

Comment:
From version 5.5.0, OMERO.insight can be installed from an Apple <z class="bold">.dmg</z> 
```suggestion
	<p class="note-text">From version 5.5.0, OMERO.insight can be installed from an Apple <z class="bold">.dmg</z> file. Running the .dmg file will mount it to your Mac. The DMG mounts in two places: on your desktop and in the Finder sidebar under your hard drive. Clicking either one of these opens the DMG file.</p>
```